connected care: add features docs (#20303)
pingcap/docs: tidb-cloud/tidb-cloud-support.md

# TiDB Cloud Support

TiDB Cloud offers tiered support plan offerings tailored to meet customers' needs. For more information, see [Connected Care Details](/tidb-cloud/connected-care-detail.md).

> **Note:**
>
> To request a [Proof of Concept (PoC)](/tidb-cloud/tidb-cloud-poc.md), a demo, or free trial credits, click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com/), and click **Contact Sales**.

## Access PingCAP Help Center

The [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals) is the central hub for TiDB Cloud users to access support services and manage support tickets.

You can access the PingCAP Help Center via <https://tidb.support.pingcap.com/servicedesk/customer/portals> directly, or through the [TiDB Cloud console](https://tidbcloud.com/) in the following ways:

> - Click **?**  the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com/), and then click **Request Support**.

Reviewer: ```suggestion
- Click **?** in the lower-right corner of the [TiDB Cloud console](https://tidbcloud.com/), and then click **Request Support**.
```

- Click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner of [TiDB Cloud console](https://tidbcloud.com/), click **Support**, and then click **Create a Support Ticket**.
- On the [**Clusters**](https://tidbcloud.com/console/clusters) page of your project, click **...** in the row of your cluster, and then select **Get Support**.
- On your cluster overview page, click **...** in the upper-right corner, and then select **Get Support**.

## Create an account or billing support ticket

To create a support ticket about account or billing issues, take the following steps:

> 1. Log into the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals), and then click [TiDB Cloud Account/Billing Support](https://tidb.support.pingcap.com/servicedesk/customer/portal/16).

Reviewer: ```suggestion
1. Log in to the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals), and then click [TiDB Cloud Account/Billing Support](https://tidb.support.pingcap.com/servicedesk/customer/portal/16).
```

2. Click **Submit a request**.
3. Fill in the following fields:

    - **Summary**: provide a brief summary of your question.
    - **TiDB Cloud Org**: select the relevant TiDB Cloud organization, if applicable.
    - **TiDB Cloud Cluster**: select the relevant TiDB Cloud cluster, if applicable.
    - **Description**: provide the details about the issue.

>     - **Severity**: estimate the business impact of the issue and choose the proper severity for it. (S1 is not applicable to billing or account issues)

Reviewer: ```suggestion
    - **Severity**: estimate the business impact of the issue and choose the proper severity for it. (S1 is not applicable to billing or account issues.)
```

4. Click **Submit**.

## Create a technical support ticket

To create a support ticket about technical issues, take the following steps:

> 1. Log into the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals), and then click [TiDB Cloud Technical Support](https://tidb.support.pingcap.com/servicedesk/customer/portal/6).

Reviewer: ```suggestion
1. Log in to the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals), and then click [TiDB Cloud Technical Support](https://tidb.support.pingcap.com/servicedesk/customer/portal/6).
```

    > **Note:**
    >

>     > The [TiDB Cloud Technical Support](https://tidb.support.pingcap.com/servicedesk/customer/portal/6) entry is only available for customers subscribed to Developer, Enterprise, or Premium support plans.

Reviewer 2: ```suggestion
    > The [TiDB Cloud Technical Support](https://tidb.support.pingcap.com/servicedesk/customer/portal/6) entry is only available for customers subscribed to **Developer**, **Enterprise**, or **Premium** support plans.
```

    - **Description**: describe the issue you encountered as detailed as possible. For example, share the exact timestamp when you encountered the issue, attach the detailed error messages and call stack of the issue, and add your troubleshooting or analysis of the issue.
    - **Severity**: estimate the business impact of the issue and choose the proper severity for it.

        | Severity | Description |
        | --- | --- |
        | S1 | Complete loss of production environmental functionality |
        | S2 | High impact on operations in production environments |
        | S3 | Non-critical database usage issues in production or non-production environments |
        | S4 | General question on how a particular feature or function performs or should be configured. An issue that has minimal impact on business and can be tolerated for a reasonable period. |

    - **Components**: select the relevant TiDB Cloud component to report the issue for, such as TiDB, TiKV, PD, or TiFlash.
    - **Affects versions**: specify the TiDB Cloud cluster version related to the issue.

4. Click **Submit**.

## View support tickets

> To view all the historical support tickets, log into the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals), click your avatar in the upper-right corner, and then click **Requests**.

Reviewer: ```suggestion
To view all the historical support tickets, log in to the [PingCAP Help Center](https://tidb.support.pingcap.com/servicedesk/customer/portals), click your avatar in the upper-right corner, and then click **Requests**.
```

## Check or upgrade your support plan

TiDB Cloud offers a free basic support plan by default. For extended services, you can upgrade to a paid plan.

To check or upgrade your support plan, perform the following steps:

1. In the [TiDB Cloud console](https://tidbcloud.com/), click <MDSvgIcon name="icon-top-organization" /> in the lower-left corner, and then click **Support** to go to the support page.

    On this page, you can find your **Current Plan**. By default, the **Basic** free plan is selected.

2. Choose your desired support plan.

    <SimpleTab>
    <div label="Upgrade to Developer or Enterprise">

    To upgrade to **Developer** or **Enterprise**:

    1. Click **Upgrade** in the **Developer** or **Enterprise** pane. An **Upgrade to Developer Plan** or **Upgrade to Enterprise Plan** page is displayed.

>     2. Check the corresponding support service information on the page. For a complete version of each support plan, see [Connected Care Details](https://docs.pingcap.com/tidbcloud/connected-care-detail) .

Reviewer: ```suggestion
    2. Check the corresponding support service information on the page. For a complete version of each support plan, see [Connected Care Details](/tidb-cloud/connected-care-detail.md).
```
